explore/src/main/scala/explore/components/FocusedStatus.scala

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) 2016-2023 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package explore.components
+
+import cats.syntax.all.*
+import explore.components.ui.ExploreStyles
+import explore.model.AppContext
+import explore.model.Focused
+import explore.model.enums.AppTab
+import japgolly.scalajs.react.*
+import japgolly.scalajs.react.vdom.html_<^.*
+import lucuma.core.model.Program
+import lucuma.react.common.ReactFnProps
+
+import scala.scalajs.LinkingInfo
+
+case class FocusedStatus(tab: AppTab, programId: Program.Id, focused: Focused)
+    extends ReactFnProps(FocusedStatus.component)
+
+/**
+ * Development utility to show the current focused state
+ */
+object FocusedStatus:
+  private type Props = FocusedStatus
+
+  private val component =
+    ScalaFnComponent
+      .withHooks[Props]
+      .useContext(AppContext.ctx)
+      .render: (props, ctx) =>
+        if (LinkingInfo.developmentMode)
```

Comment:
I would put the `if (LinkingInfo.developmentMode)` at use sites. That will tree-shake the whole component off in production.